unit_tests/utilities/test_field_plane_averaging.cpp

```diff
@@ -184,11 +183,9 @@ void add_periodic(
     auto dx = geom.CellSizeArray();
 
     amrex::ParallelFor(bx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
-        amrex::Real x[3];
-
-        x[0] = xlo[0] + (i + 0.5) * dx[0];
-        x[1] = xlo[1] + (j + 0.5) * dx[1];
-        x[2] = xlo[2] + (k + 0.5) * dx[2];
+        const amrex::GpuArray<amrex::Real, 3> x = {
```

Comment:
It would be nicer and more readable to format it as before with each index in its own line. Did the clang-format change it to this?

Reply:
yeah clang format did it.